backend/fms_core/template_importer/sheet_data.py

```diff
@@ -40,25 +40,32 @@ def prepare_rows(self):
         for row_id in data_row_ids_range(self.header_row_nb + 1, self.dataframe):
             row_data = self.dataframe.iloc[row_id]
             self.rows.append(row_data)
-
-            row_repr = f"#{row_id + 1}"
+            row = row_id + 1
+            row_repr = f"#{row}"
+            row_data = panda_values_to_str_list(row_data)
```

Comment:
Be careful with reallocation of variables ... the python language typically uses the reference when passing data around so you might end up breaking the self.rows by doing this here. I would instead create another variable at that point.